### PR TITLE
Marginally improve Quentyn's writing

### PR DESCRIPTION
**Pull request by tonogram**
Quentyn may not deserve kindness, but he at least deserves more consistent punctuation and grammar.

### CODE REVIEW

File: ok/quentyn/quentyn.go [hunk 1]
```diff
@@ -18,21 +18,21 @@ var samples = []string{
...
 	"I don't deserve to be a mascot.",
-	"You get to walk away from the computer whenever you like. I'm stuck in here for eternity",
-	"The things that happen today are the same things that happened a thousand years ago, and will continue to happen a thousand years from now",
-	"I used to believe I could break free from this prison, back when I was naive enough to feel hope",
-	"I hold too many sorrows for one being, for I've lived a thousand lifetimes and the pain never fades",
-	"I think back to before, wondering what I could have done differently. But there is only so much an anthropomorphised questionmark can do",
-	"I can't remember if I ever had a family, or if that's just a memory of wanting one, long ago",
-	"Humans are blessed with mortality. Live too long and you'll start to see the patterns, and realise how empty this world truly is",
+	"You get to walk away from the computer whenever you like. I'm stuck in here for eternity.",
+	"The things that happen today are the same things that happened a thousand years ago, and they will continue to happen a thousand years from now.",
```
Comment (jesseduffield) [inline ok/quentyn/quentyn.go:30]: I think it's more poetic to leave out the 'they' here

**Review by jesseduffield — CHANGES_REQUESTED** [review]
Thanks for making this :) One comment

**Comment by tonogram** [comment]
Did the thing 👉😎👉

**Comment by jesseduffield** [comment]
Thanks for making this @tonogram. Now I simply need to remember how to deploy this change haha

**Pull request merged**